assets/src/data/model/status/constants.js

```diff
@@ -7,46 +12,67 @@ export const STATUS_TYPE_PAYMENT = 'payment';
 export const STATUS_TYPE_REGISTRATION = 'registration';
 export const STATUS_TYPE_TRANSACTION = 'transaction';
 // email
-export const STATUS_ID_EMAIL_DRAFT = 'EDR';
-export const STATUS_ID_EMAIL_SENT = 'ESN';
-export const STATUS_ID_EMAIL_EXPIRED = 'EXP';
+export const emailStatus = {
```

Comment:
Might be better to give this a slightly more accurate name since `emailStatus` sounds singular. 
Maybe `emailStatusArray` or `allEmailStatus` or something like that???

This would also leave the `emailStatus` name available for client code to use to indicate a singular status which would be the natural choice when trying to name such a variable imho.

Same goes for the rest of these constants:

- `eventStatusArray` or `allEventStatus` or ???
- `messageStatusArray` or `allMessageStatus` or ???
- `paymentStatuArrays` or `allPaymentStatus` or ???
- `registrationStatusArray` or `allRegistrationStatus` or ???
- `transactionStatusArray` or `allTransactionStatus` or ???

Reply:
I disagree, this is not an array it is an object.  When implemented in code the usage will look something like:

```
emailStatus.SENT
```

If there's cases where client code may want to name a variable to represent status the import can assign the constant to a different name a-la:

```
import { emailStatus as someOtherName }
```

So to sum up, I picked the name because its _typical_ usage makes more sense with that label.

Reply:
doh... right.. that's an object :facepalm: